### PR TITLE
style: format files with prettier

### PR DESCRIPTION
**Pull request by rafaelss95**

### CODE REVIEW

File: src/rules/effects/use-effects-lifecycle-interface.ts [hunk 1]
```diff
@@ -59,10 +59,8 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
 
         if (interfaces.includes(interfaceName)) return
 
-        const {
-          implementsNodeReplace,
-          implementsTextReplace,
-        } = getImplementsSchemaFixer(classDeclaration, interfaceName)
+        const { implementsNodeReplace, implementsTextReplace } =
+          getImplementsSchemaFixer(classDeclaration, interfaceName)
```
Comment (rafaelss95) [inline src/rules/effects/use-effects-lifecycle-interface.ts:63]: I was seeing the following report by `prettier` (here and in /guards):

Prettier report

File: src/utils/helper-functions/guards.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import { AST_NODE_TYPES, TSESTree } from '@typescript-eslint/experimental-utils'
 import ts from 'typescript'
 
-const isNodeOfType = <NodeType extends AST_NODE_TYPES>(nodeType: NodeType) => (
-  node: TSESTree.Node,
-): node is TSESTree.Node & { type: NodeType } => node.type === nodeType
+const isNodeOfType =
+  <NodeType extends AST_NODE_TYPES>(nodeType: NodeType) =>
+  (node: TSESTree.Node): node is TSESTree.Node & { type: NodeType } =>
+    node.type === nodeType
```
Comment (rafaelss95) [inline src/utils/helper-functions/guards.ts:7]: I was seeing the following report by `prettier` (here and in /use-effects-lifecycle-interface.ts):

Prettier report

File: src/utils/helper-functions/guards.ts [hunk 3]
```diff
@@ -30,6 +31,5 @@ export const isObjectExpression = isNodeOfType(AST_NODE_TYPES.ObjectExpression)
 export const isProperty = isNodeOfType(AST_NODE_TYPES.Property)
 
 export function isTypeReference(type: ts.Type): type is ts.TypeReference {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  return Boolean((type as any).target)
+  return type.hasOwnProperty('target')
```
Comment (rafaelss95) [inline src/utils/helper-functions/guards.ts:34]: Taking into consideration the fact that I was messing up this file, I changed the implementation here so we can remove the typecast.

**Review by timdeschryver — APPROVED** [review]
LGTM.
Do you think we need a check in our CI that verifies if prettier has run?

File: src/utils/helper-functions/guards.ts (hunk 3 above)
Comment (timdeschryver) [reply src/utils/helper-functions/guards.ts:34]: Good catch!

**Comment by rafaelss95** [comment]
> LGTM.
> Do you think we need a check in our CI that verifies if prettier has run?

It'd be great 😄

**Pull request merged**